libiop/tests/bcs/test_merkle_tree.cpp

```diff
@@ -12,51 +13,62 @@
 
 namespace libiop {
 
+using std::size_t;
+
 template< bool B, class T = void >
 using enable_if_t = typename libff::enable_if<B,T>::type;
+
 // Binary hash type
 template<typename FieldT, typename hash_type,
     enable_if_t<!std::is_same<FieldT, hash_type>::value, int> = 42>
 merkle_tree<FieldT, hash_type> new_MT(
-    const std::size_t size, const std::size_t digest_len_bytes, const bool make_zk,
-    const std::size_t security_parameter)
+    const size_t size, const size_t digest_len_bytes, const bool make_zk,
+    const size_t security_parameter, const size_t cap_size=2)
 {
     return merkle_tree<FieldT, binary_hash_digest>(
         size,
         std::make_shared<blake2b_leafhash<FieldT>>(security_parameter),
         blake2b_two_to_one_hash,
+        blake2b_many_to_one_hash,
         digest_len_bytes,
         make_zk,
-        security_parameter);
+        security_parameter,
+        cap_size);
 }
+
 template<typename FieldT, typename hash_type,
     enable_if_t<std::is_same<FieldT, hash_type>::value, int> = 42>
 merkle_tree<FieldT, hash_type> new_MT(
-    const std::size_t size, const std::size_t digest_len_bytes, const bool make_zk,
-    const std::size_t security_parameter)
+    const size_t size, const size_t digest_len_bytes, const bool make_zk,
+    const size_t security_parameter, const size_t cap_size=2)
 {
     return merkle_tree<FieldT, FieldT>(
         size,
         std::make_shared<dummy_algebraic_leafhash<FieldT>>(),
         dummy_algebraic_two_to_one_hash<FieldT>,
+        dummy_algebraic_cap_hash<FieldT>,
         digest_len_bytes,
         make_zk,
-        security_parameter);
+        security_parameter,
+        cap_size);
 }
 
+/** Constructs a merkle tree with leaf size 2. Generates and verifies membership proofs for
```

Comment:
What is the convention for ** or * in libiop?

Reply:
I don't know if there's an existing convention, the one I use is /** if it's documentation that the user might want to see on hover, and /* if it's a multiline comment that's only useful to see in the location where it's added (// if it's a single line comment)

Reply:
Or rather, that's the philosophy I have now. Looks like I didn't stick to it at the time

Reply:
Considered this one resolved.